src/openrct2/ride/Ride.cpp

```diff
@@ -7658,12 +7657,10 @@ StationObject* ride_get_station_object(const Ride* ride)
     return static_cast<StationObject*>(objManager.GetLoadedObject(OBJECT_TYPE_STATION, ride->entrance_style));
 }
 
-LocationXY16 ride_get_rotated_coords(int16_t x, int16_t y, int16_t z)
+ScreenCoordsXY ride_get_rotated_coords(CoordsXYZ coords3d)
```

Comment:
This function is now just a wrapper for `translate_3d_to_2d_with_z`. Its a little pointless.